CoreFoundation/Base.subproj/CoreFoundation_Prefix.h

```diff
@@ -68,6 +68,33 @@ typedef char * Class;
 #include <pthread.h>
 #endif
 
+#if TARGET_OS_WIN32
+#define BOOL WINDOWS_BOOL
+
+#define MAXPATHLEN MAX_PATH
+#undef MAX_PATH
+#undef INVALID_HANDLE_VALUE
+
+#define WIN32_LEAN_AND_MEAN
+
+#ifndef WINVER
+#define WINVER  0x0601
+#endif
+
+#ifndef _WIN32_WINNT
+#define _WIN32_WINNT 0x0601
+#endif
+
+// The order of these includes is important
+#define FD_SETSIZE 1024
+#include <winsock2.h>
+#include <windows.h>
+#include <time.h>
```

Comment:
Don't forget time.h is included adjacent to mach_absolute_time.

Reply:
That needed to be shuffled due to the inclusion of `Windows.h`.

Reply:
What I mean is: can you omit it here again, considering now time.h is included a second time below windows.h, or is there some other reason preventing that?

Reply:
oh, I see; I'll do that as a follow up.  The time.h was moved since it was in the block.